File: 1676B - Equal Candies.java
import java.util.Scanner;
public class Main{
    public static void main(String [] lol){
        Scanner scanner = new Scanner(System.in);
        int t= scanner.nextInt();
        for(int i=0;i<t;i++){
            int n= scanner.nextInt(); // number of boxes
            int [] candies = new int[n];  // input for arrays
            for(int j=0;j<n;j++){
                candies[j]= scanner.nextInt();
                // read the input for the aarays
            }
            int minCandies= 0;
            for (int candy: candies){ min=Math.min(minCandies,candy);}
            int totalCandiesToEat =0;
            for(int candy : candies){ totalCandiesToEat += candy - minCandies;}
            System.out.println(totalCandiesToEat);
        }
        scanner.close();
    }
}

File: 1676B - Equal Candies.py
t= int(intput())
for _ in range(t):
    n = int(intput())
    candies= list(map(int,input().split()))
    min_candies= min(candies)
    candies_to_eat= [c- min_candies for c in candies]
    total_candies_to_eat= sum(candies_to_eat)
    print(total_candies_to_eat)
